praetorian_cli/ui/console/commands/tools.py

```
@@ -543,3 +543,172 @@ def _cmd_capabilities(self, args):
+            if name in installed:
+                status = f'[success]{ver["version"]}[/success]' if ver else '[success]yes[/success]'
+            else:
+                status = '[dim]—[/dim]'
+            desc = r.get('description', '')
+            if len(desc) > 50:
+                desc = desc[:49] + '…'
+            table.add_row(str(i), name, r.get('category', ''), status, desc)
+            self._module_list.append(name)
+
+        self.console.print(table)
+        self.console.print(f'\n[dim]Use "info <name>" for details or "install <name>" to install.[/dim]')
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)